### ISSUE TITLE
InstaBoost: Boosting Instance Segmentation via Probability Map Guided Copy-Pasting

### ISSUE THREAD

**Issue by AkiraTOSEI** [AkiraTOSEI/ML_papers#111]
# TL;DR
A study to implement segmentation data augmentations by cutting and pasting objects. By creating a heat map so that it is pasted into the similar place as original, they are able to perform plausible data augmentations.
![Figure 1  An example of random InstaBoost and appearance con-](https://user-images.githubusercontent.com/24524018/93177726-340dbc00-f76e-11ea-8914-23c0bf9f9844.png)

# Paper URL
https://arxiv.org/abs/1908.07801

# Submission Dates(yyyy/mm/dd)
2019/08/21

# Authors and institutions
Hao-Shu Fang, Jianhua Sun, Runzhong Wang, Minghao Gou, Yong-Lu Li, Cewu Lu
    - Shanghai Jiao Tong University, China

# Methods

# Results

# Comments
